mesos_slave/datadog_checks/mesos_slave/mesos_slave.py

```diff
@@ -93,22 +93,33 @@ class MesosSlave(AgentCheck):
         'slave/valid_status_updates': ('mesos.slave.valid_status_updates', GAUGE),
     }
 
+    HTTP_CONFIG_REMAPPER = {'disable_ssl_validation': {'name': 'tls_verify', 'invert': True}}
+
     def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
         self.cluster_name = None
         for instance in instances or []:
             url = instance.get('url', '')
             parsed_url = urlparse(url)
-            ssl_verify = not _is_affirmative(instance.get('disable_ssl_validation', False))
-            if not ssl_verify and parsed_url.scheme == 'https':
-                self.log.warning('Skipping SSL cert validation for %s based on configuration.' % url)
-
-    def _get_json(self, url, timeout, verify, tags=None):
+            if self.http.options['tls_verify'] and parsed_url.scheme == 'https':
+                self.log.warning('Skipping TLS cert validation for %s based on configuration.' % url)
+
+            # `default_timeout` config option will be removed with Agent 5
+            self.http.options['timeout'] = (
```

Comment:
Same as in the other PR, the `connect` and `receive` timeout parameters are not used